js/index.js

```diff
@@ -58,18 +62,22 @@ webix.ready(function(){
     minWidth:400,
     scheme:{
         $init:function(obj){
+            obj.categoryId = getRandomInt(1,4);
+            obj.rank = this.count();//Like variant
```

Comment:
looks strange. $init applied to loaded data as well, so you just corrupt the initial data. you can check `if(!obj.rank)` for example. but why this line is necessary at all?

Reply:
In situation, when you add new element, "rank" cell will be empty without this line. (obj.rank = this.count())